### PR TITLE
0.12.0 blog post

### PR DESCRIPTION
**Pull request by gep13**
@devlead @patriksvensson can I get you to eye ball this?

### CODE REVIEW

File: src/Cake.Web/App_Data/blog/2016-05-25-cake-v0.12.0-released.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- [__#895__](https://github.com/cake-build/cake/issues/895) Consider adding developmentDependency to nuspec for Cake.Common
+- [__#889__](https://github.com/cake-build/cake/pull/889) Specify culture for Roslyn debug string formats
+- [__#881__](https://github.com/cake-build/cake/issues/881) Add cake-build build.cake NuGet restore retry handling
+- [__#860__](https://github.com/cake-build/cake/issues/860) During the publishing cycle, continue with each step, and error at end if there is a problem
+- [__#859__](https://github.com/cake-build/cake/issues/859) Add all artifacts to AppVeyor during publishing cycle
+- [__#752__](https://github.com/cake-build/cake/issues/752) Add register and ReturnTargetCodeOffset option in opencover
+- [__#736__](https://github.com/cake-build/cake/issues/736) Unified tool resolution
+- [__#627__](https://github.com/cake-build/cake/issues/627) Drop DNU/DNX support in favor of dotnet CLI
+
+__Documentations__
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/blog/2016-05-25-cake-v0.12.0-released.md]: Can we change this to `__Documentation__`?

**Comment by patriksvensson** [comment]
Looks good to me! :+1:

**Comment by devlead** [comment]
LGTM 👍

**Comment by gep13** [comment]
@patriksvensson corrected the heading :smile: I need to get the PR that you sent to GitReleaseManager pulled in so that this can be overridden :+1:

**Comment by patriksvensson** [comment]
@gep13 Yeah, but only if you think it's worth being pulled in :smile:

**Comment by gep13** [comment]
>@patriksvensson said...
@gep13 Yeah, but only if you think it's worth being pulled in :smile:

Oh, it is, I just need to get it pulled in :smile:

**Pull request merged**